src/ci/scripts/should-skip-this.sh

```diff
@@ -26,6 +26,11 @@ elif ! git diff --quiet "$BASE_COMMIT" -- src/tools/clippy src/tools/rustfmt; th
     # There is not an easy blanket search for subtrees. For now, manually list
     # the subtrees.
     echo "Executing the job since clippy or rustfmt subtree was updated"
+elif (git diff --name-only HEAD^ | grep --quiet src/test/rustdoc-gui) || \
+     (git diff --name-only HEAD^ | grep --quiet src/librustdoc) || \
+     (git diff --name-only HEAD^ | grep --quiet src/tools/rustdoc-gui); then
```

Comment:
We've since changed to using the git diff commands with BASE_COMMIT, like above.

Reply:
:+1: